e2e/helpers/pages/admin/posts/posts-page.ts

```diff
@@ -13,13 +24,75 @@ export class PostsPage extends AdminPage {
         this.postsList = page.getByTestId('posts-list');
         this.postsListItem = this.postsList.getByTestId('posts-list-item');
         this.newPostButton = page.getByRole('link', {name: 'New post'});
+
+        this.typeFilter = page.getByRole('button', {name: 'Type filter'});
+        this.visibilityFilter = page.getByRole('button', {name: 'Visibility filter'});
+        this.authorFilter = page.getByRole('button', {name: 'Author filter'});
+        this.tagFilter = page.getByRole('button', {name: 'Tag filter'});
+        this.orderFilter = page.getByRole('button', {name: 'Sort filter'});
+
+        this.saveViewButton = page.getByRole('button', {name: /save as view/i});
+        this.editViewButton = page.getByRole('button', {name: /edit current view/i});
+
+        this.pageTitle = page.getByRole('heading', {level: 2});
     }
 
     getPostByTitle(title: string): Locator {
         return this.postsListItem.filter({has: this.page.getByRole('heading', {name: title, exact: true, level: 3})});
     }
 
+    async waitForPostsToLoad(): Promise<void> {
+        await Promise.race([
```

Comment:
`nit:` maybe we could describe the intention of this wait function with a comment, so it's clear why we handle the race condition and silence rejections

Reply:
`nit` and maybe make it even clearer with named locators:

```
await Promise.race([
    firstPost.waitFor({state: 'visible'}).catch(() => {}),
    emptyState.waitFor({state: 'visible'}).catch(() => {})
]);
```